airflow/dags/disambiguation/inventor_assignee_disambiguation.py

```diff
@@ -57,7 +57,7 @@ class SQLTemplatedPythonOperator(PythonOperator):
 disambiguation = DAG(
     dag_id='inventor_assignee_disambiguation',
     default_args=default_args,
-    description='Perform inventor & assignee disambiguation',
+    description='Perform inventor, assignee, & location disambiguation',
```

Comment:
Are we doing anything location related in this dag? We should probably clean those up and remove this description if not

Reply:
Yes we are still creating the canonical_locations table quarterly